lib/app/modules/us_core_r4/us_core_practitionerrole_sequence.rb

```diff
@@ -49,8 +49,9 @@ def validate_resource_item(resource, property, value)
         @client.set_no_auth
         skip 'Could not verify this functionality when bearer token is not set' if @instance.token.blank?
 
-        specialty_val = @practitionerrole&.specialty&.coding&.first&.code
+        specialty_val = resolve_element_from_path(@practitionerrole, 'specialty.coding.code')
```

Comment:
Where are these changes coming from? I don't see them reflected in the generator diff.

Reply:
That line should've been changed in a previous PR, but for some reason it didn't git committed. Running the generator again just added the changes back in.